macao-sdk-mirror/src/commonMain/kotlin/com/macaosoftware/plugin/util/MacaoResult.kt

```diff
@@ -11,3 +11,6 @@ sealed class MacaoResult2<out S, out E> {
 }
 
 interface MacaoError
+sealed class MacaoErrors : MacaoError{
```

Comment:
In i=this module a sealed class doesn't help much. Sealed class only work in the same module they are declared. Most of the time the errors will be declared in plugin modules so it wouldn't be used so much. Feel free to remove it.